pkg/virt-launcher/virtwrap/api/converter.go

```diff
@@ -495,38 +499,167 @@ func Convert_v1_VirtualMachine_To_api_Domain(vmi *v1.VirtualMachineInstance, dom
 		interfaceType = vmi.ObjectMeta.Annotations[v1.InterfaceModel]
 	}
 
-	findNetwork := func(nets []v1.Network, name string) (*v1.Network, error) {
-		for _, net := range nets {
-			if net.Name == name {
-				return &net, nil
-			}
-		}
-		return nil, fmt.Errorf("failed to find network %s", name)
+	networks := map[string]*v1.Network{}
+	for _, network := range vmi.Spec.Networks {
```

Comment:
I can't see a clear benefit for it being a map or a loop - we still have to loop the same amount of times anyway.